packages/api/tests/endpoints/test-rules.js

```diff
@@ -1183,16 +1419,21 @@ test.serial('put() creates the same updated Kinesis rule in Dynamo/PostgreSQL/El
   const updatedRule = await ruleModel.get({ name: updateRule.name });
   const updatedPgRule = await t.context.rulePgModel
     .get(t.context.testKnex, { name: updateRule.name });
-  const updatedEsRecord = await t.context.esRulesClient.get(
+  const updatedEsRule = await t.context.esRulesClient.get(
     originalDynamoRule.name
   );
-  t.truthy(updatedRule.rule.arn);
-  t.truthy(updatedRule.rule.logEventArn);
-  t.truthy(updatedPgRule.arn);
-  t.truthy(updatedPgRule.log_event_arn);
+
+  t.truthy(originalDynamoRule.rule.arn);
```

Comment:
Is this block a duplication of L1393 above?   Unless I'm misunderstanding something I think these can be removed.

Reply:
It was and in fact needed to be updated to test what I want:

88c3502310f14be77f3168d5532e3745dd0e8313